src/WrapperApp/components/Simulation/SimulationPanel.tsx

```diff
@@ -159,38 +159,6 @@ export default function SimulationPanel({
 		setInputFiles(inputFiles);
 	};
 
-	/**
```

Comment:
Why is this code removed ?

Reply:
old function for running simulations. Replaced with sendSimulationRequest